src/main/java/com/redhat/devtools/intellij/tektoncd/ui/component/StartTaskDialog.java

```diff
@@ -0,0 +1,459 @@
+/*******************************************************************************
```

Comment:
I suppose the `component` sub package comes from the OpenShift plugin but I think this should be renamed as `task` ?